go.mod

```diff
@@ -71,7 +71,7 @@ require (
 	github.com/mattn/go-isatty v0.0.20
 	github.com/olekukonko/tablewriter v0.0.5
 	github.com/opencontainers/image-spec v1.0.2
-	github.com/ory/dockertest v3.3.5+incompatible
+	github.com/ory/dockertest/v3 v3.10.0
```

Comment:
Nice! Any other dependencies with version v2 or above that we can be switched to this format?

Reply:
`cupaloy` (for Flink) and `stripe-go` (for payment).

I'm inclined to leave `stripe-go` alone for now because I don't want to make any changes to the payment code unless we need to.

I'll go ahead and update `cupaloy` and run the vulnerability scan again though.

Reply:
> I'm inclined to leave stripe-go alone for now because I don't want to make any changes to the payment code unless we need to.

I say do it... the code itself shouldn't be any different, just the format of the dependency